TRANSFORMERS/VECTOR_MANIPULATION/DECIMATE_VECTOR/DECIMATE_VECTOR.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from flojoy import flojoy, Vector
+
+
+@flojoy
+def DECIMATE_VECTOR(
+    default: Vector,
+    factor: int = 1,
+) -> Vector:
+    """The DECIMATE_VECTOR node returns the input vector where
```

Comment:
This isn't what decimate does. Decimate is reducing the number of points by a certain factor. https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.decimate.html